fetch_data.py

```diff
@@ -93,7 +93,8 @@ async def make_get_request(url, session):
     Returns:
         Query result, deocded differently depending on encoding argument.
     """
-    resp = await session.request(method="GET", url=url)
+    cache_header = {"Cache-Control": 'max-age=7776000, s-maxage=7776000'}
```

Comment:
Should `s-maxage=7776000` be removed from here, too? I don't know one way or the other, just curious!